lib/Bio/EnsEMBL/DataCheck/index.json

```diff
@@ -538,6 +538,16 @@
       "name" : "EpigenomeHasSegmentationFile",
       "package_name" : "Bio::EnsEMBL::DataCheck::Checks::EpigenomeHasSegmentationFile"
    },
+   "EvidenceFreeGO" : {
+      "datacheck_type" : "critical",
+      "description" : "All GO xrefs have an evidence",
+      "groups" : [
+         "compare_core",
```

Comment:
```suggestion
```

Reply:
Thanks, that was a copy paste, erm... Fixed in commit.